module/Application/view/layout/layout.phtml

```diff
@@ -9,54 +9,64 @@
         ->appendName('viewport', 'width=device-width, initial-scale=1.0')
     ?>
 
+    <!-- Display alternate language links-->
+    <?php
+    if (isset($_SERVER['REQUEST_URI'])) {
+        $this->headLink([
+            'rel' => 'alternate', 'type' => 'text/html', 'hreflang' => 'en',
+            'href' => 'https://gewis.nl/lang/en/' . ltrim($_SERVER['REQUEST_URI'], '/')
+        ]);
+        $this->headLink([
+            'rel' => 'alternate', 'type' => 'text/html', 'hreflang' => 'nl',
+            'href' => 'https://gewis.nl/lang/nl/' . ltrim($_SERVER['REQUEST_URI'], '/')
+        ]);
+    }
+    ?>
+
     <!-- Preload fonts-->
-    <?= $this->headLink([
+    <?php $this->headLink([
         'as' => 'font', 'rel' => 'preload', 'type' => 'font/woff2', 'crossorigin' => 'anonymous',
         'href' => '/fonts/raleway/raleway-v22-latin-500.woff2'
     ]); ?>
-    <?= $this->headLink([
+    <?php $this->headLink([
         'as' => 'font', 'rel' => 'preload', 'type' => 'font/woff2', 'crossorigin' => 'anonymous',
         'href' => '/fonts/raleway/raleway-v22-latin-600.woff2'
     ]); ?>
-    <?= $this->headLink([
+    <?php $this->headLink([
         'as' => 'font', 'rel' => 'preload', 'type' => 'font/woff2', 'crossorigin' => 'anonymous',
         'href' => '/fonts/raleway/raleway-v22-latin-regular.woff2'
     ]); ?>
-    <?= $this->headLink([
+    <?php $this->headLink([
         'as' => 'font', 'rel' => 'preload', 'type' => 'font/woff2', 'crossorigin' => 'anonymous',
         'href' => '/fonts/lato/lato-v22-latin-700.woff2'
     ]); ?>
-    <?= $this->headLink([
+    <?php $this->headLink([
         'as' => 'font', 'rel' => 'preload', 'type' => 'font/woff2', 'crossorigin' => 'anonymous',
         'href' => '/fonts/lato/lato-v22-latin-regular.woff2'
     ]); ?>
-    <?= $this->headLink([
+    <?php $this->headLink([
         'as' => 'font', 'rel' => 'preload', 'type' => 'font/woff2', 'crossorigin' => 'anonymous',
         'href' => '/fonts/fontawesome/fa-solid-900.woff2'
     ]); ?>
-    <?= $this->headLink([
+    <?php $this->headLink([
         'as' => 'font', 'rel' => 'preload', 'type' => 'font/woff2', 'crossorigin' => 'anonymous',
         'href' => '/fonts/fontawesome/fa-brands-400.woff2'
     ]); ?>
-    <?= $this->headLink([
+    <?php $this->headLink([
         'as' => 'font', 'rel' => 'preload', 'type' => 'font/ttf', 'crossorigin' => 'anonymous',
         'href' => '/fonts/gewisicons/gewisicons.ttf?37l19n='
     ]); ?>
-    <?= $this->headLink([
+    <?php $this->headLink([
         'as' => 'image', 'rel' => 'preload', 'type' => 'image/svg+xml', 'crossorigin' => 'anonymous',
         'href' => '/img/nl.svg'
     ]); ?>
-    <?= $this->headLink([
+    <?php $this->headLink([
         'as' => 'image', 'rel' => 'preload', 'type' => 'image/svg+xml', 'crossorigin' => 'anonymous',
         'href' => '/img/en.svg'
     ]); ?>
```

Comment:
Not an issue, but could probably do with one `<?php ?>` for all of them (and possibly chain them too.)